testutils/src/main/scala/za/co/absa/testutils/RandomDataGenerator.scala

```diff
@@ -13,20 +13,14 @@
  * limitations under the License.
  */
 
-package za.co.absa
+package za.co.absa.testutils
+
+import java.io.{BufferedWriter, File, FileWriter}
 
-import java.io._
 import org.joda.time.DateTime
 import org.joda.time.format.{DateTimeFormat, DateTimeFormatter}
-import scala.util.Random
-import Numeric.Implicits._
 
-/* Generate data for testing
```

Comment:
Why remove the comment? I'd make it a scaladoc (i.e. `/**` rather than `/*`)

Reply:
I don't remember deleting it. Is it possible this is caused by automatic refractor?